core/v1/types.go

```diff
@@ -3589,6 +3589,9 @@ type NodeSpec struct {
 	// The DynamicKubeletConfig feature gate must be enabled for the Kubelet to use this field
 	// +optional
 	ConfigSource *NodeConfigSource `json:"configSource,omitempty" protobuf:"bytes,6,opt,name=configSource"`
+	// Message describes reason why node is marked as unschedulable or notready. By default, message is empty.
```

Comment:
The code changes to `k8s.io/api` do not go in this repo. This repo is not authoritative and the code in this repo is copied from https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/api.

Please submit it as a PR to the main `kubernetes/kubernetes` repo and make the changes here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go. They will be later synced to here.

Reply:
@nikhita Thanks for the correctness. 👍